### DISCUSSION TITLE
Import unknown libraries

### DISCUSSION THREAD

**Discussion by Xylar52** [pybricks/support#981] [Q&A]
In the Spike Prime app, there is a library pre-installed called spike and spike.control. I was wondering if it was possible to import it in a pybricks file so we would have even more to choose from. 

![Capture d’écran 2023-03-09 220204](https://user-images.githubusercontent.com/90038318/224157597-d47054d1-8e59-4885-9973-215e3c89c71c.png)

**Comment by dlech** [comment]
`pybricks.tools` is the equivalent in Pybricks. 

| Pybricks | LEGO |
|-|-|
| `wait(... / 1000)` | `wait_for_seconds(...)` |
| `while not ...: pass` | `wait_until(...)` |
| `StopWatch()` | `Timer()` |
